dynamiqs/utils/jax_utils.py

```diff
@@ -95,3 +81,45 @@ def to_qutip(x: ArrayLike, dims: tuple[int, ...] | None = None) -> Qobj | list[Q
         elif isop(x):  # [[3], [3]] or for composite systems [[3, 4], [3, 4]]
             dims = [dims, dims]
         return Qobj(x, dims=dims)
+
+
+def device(backend: Literal['cpu', 'gpu', 'tpu']):
```

Comment:
WDYT of using a more explicit name, e.g. `dq.set_device` and `dq.set_precision`? IMO `dq.device` rather refers to an actual device, and not to the action of setting one. It won't be super used functions so I think it's ok that the name is a bit longer.